### PR TITLE
Fix: Style variable instead of prop for Input

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Refactor `Input` and remove the prop `withErrorMessage`.

# Changes

* Use a css variable instead of a prop to change Input style.

# Tests

No test changes.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker PR removing Input prop `withErrorMessage`

**Review by bitdivine — APPROVED** [review]

File: frontend/svelte/src/lib/components/ui/Input.svelte [hunk 1]
```diff
@@ -60,7 +59,10 @@
   .input-block {
     position: relative;
 
-    margin: var(--padding-2x) 0;
+    margin-top: var(--padding-2x);
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/ui/Input.svelte:62]: adding only the new property `margin: var(--padding-2x) 0 var(--input-margin-bottom, var(--padding-2x));` would work too right? just for my know-how, splitting is fine

**Review by peterpeterparker — APPROVED** [review]
Thanks

File: frontend/svelte/src/lib/components/ui/Input.svelte (hunk 1 above)
Comment (lmuntaner) [reply frontend/svelte/src/lib/components/ui/Input.svelte:62]: Yes, it would work, but I found that harder to read.

**Pull request merged**